quickstart-mongodbatlas/beta-launch.md

```diff
@@ -0,0 +1,96 @@
+
+![atlas-cfn](https://gist.githubusercontent.com/jasonmimick/68f402c378ed364ea1684fda1a7ef5d2/raw/3094b7a2f77d4a8d3e0a8acf4a876844acb39685/atlas-cfn.png "atlas-cfn") MongoDB Atlas Cloud Formation Custom Resources
+=====
+
+# (Pre)-Beta Launch "README"
```

Comment:
Probably remove the pre since all else uses beta unless you mean specifically of the quickstart.  If that's the case I'd just include that in the header, e.g. : 
 # (Pre)-Beta Launch - MongoDB Atlas CloudFormation Quickstart - README